File: test_contest/solutions/phantom.py

n = int(input())

for testCase in range(n):

    # Print the appropriate response the appropriate number of times.
    cmd, t = input().split()
    t = int(t)

    result = {'RING':(t*['Hey, Superman!']), 
              'BANG':(t*['Are you jelly?'])}

    # Print the appropriate response t times based on input
    for reply in result[cmd]:
        print(reply)

File: test_contest/solutions/phantom.cpp
#include <iostream>
#include <string.h>

using namespace std;

int main()
{
	int T;
	cin >> T;
	for(int t = 1; t<=T; t++)
	{
		string s;
		int times;
		cin >> s >> times;
		int isBang = strcmp((const char*)s.c_str(), "RING");
		for(int i = 0; i<times; i++)
		{
			cout << (isBang ? "Are you jelly?" : "Hey, Superman!") << endl;
		}
	}
}
